tsconfig.base.json

```diff
@@ -9,7 +9,7 @@
     "module": "esnext",
     "moduleResolution": "node",
     "strict": true,
-    "strictPropertyInitialization": true,
+    "strictPropertyInitialization": false,
```

Comment:
Was ist das hier?
An dieser Stelle haben wir eigentlich noch nie etwas geändert - ich würde das nur machen, wenn es wirklich notwendig ist und das sehe ich im Moment nicht. Alles, was `strict` ist, ist potenziell besser weil es uns in der Zukunft Probleme verhindert.